backend/news-service/src/RouteHandler.js

```diff
@@ -31,10 +34,16 @@ function fillTimeSpan(request, response) {
 // /api/request/timespan
 function requestTimespan(request, response) {
   const { requestId, requestedResource } = request.body;
-  logger.debug(`Got a request for: ${requestedResource}`);
+  logger.debug(`Got a request for: ${JSON.stringify(requestedResource)}`);
   if (requestedResource) {
-    response.sendStatus(200);
+    requestedResource.from = new Date(requestedResource.from);
+    requestedResource.until = new Date(requestedResource.until);
+    if (requestedResource.until >= Date.now()) {
+      response.setStatus(400).send('You have requested things newer than the current time, plz dont.');
```

Comment:
Some inconsistent naming:

you write `response` instead of `res` and `req` instead of `request` 

Suggestion:

either always use `res` and `req` or always `response` and `request`

If you feel like it :)

Reply:
No ty